invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImageGrid.tsx

```diff
@@ -48,6 +58,10 @@ const GalleryImageGrid = () => {
   const { currentViewTotal } = useBoardTotal(selectedBoardId);
   const queryArgs = useAppSelector(selectListImagesBaseQueryArgs);
 
+  const lastSingleSelectionImage = useAppSelector(selector);
+
+  const imgRef = useRef<VirtuosoGridHandle>(null);
```

Comment:
Suggest a `virtuosoRef` as a clearer name for this

Reply:
fixed with latest commit!